egs/librispeech/ASR/transducer_emformer/emformer.py

```diff
@@ -27,6 +27,66 @@
 LOG_EPSILON = math.log(1e-10)
 
 
+def unstack_states(
```

Comment:
`unstack_states` and `stack_states` take the advantage that Emformer also outputs the states for the next chunk.

`unstack_states` is useful to split the states for a batch into a state list corresponding to separate utterances in the batch, which can be later stacked together using `stack_states`.